pkg/daemon/ceph/osd/volume.go

```diff
@@ -162,7 +163,7 @@ func (a *OsdAgent) configureCVDevices(context *clusterd.Context, devices *Device
 			// And thus won't be a block device
 			//
 			// The same goes the metadata block device which is stored in /srv
-			if !strings.HasPrefix(dev, "/mnt") && !strings.HasPrefix(dev, "/srv") {
+			if !strings.HasPrefix(dev, "/mnt") && !strings.HasPrefix(dev, "/srv") && !strings.HasPrefix(dev, "/srv2") {
```

Comment:
Where did the `/srv2` prefix come from? Won't `/srv2` already be covered by the check for the `/srv` prefix?

Reply:
I changed the mount point of wal device to /wal, instead of previous /srv2 or /srvwal, since currently the prefix name decide the device type (/mnt : block, /srv: metadata, /wal: wal) instead of by index.